### PR TITLE
[#175146179] Fix user-identity spec and add logout spec

### PR DESCRIPTION
**Pull request by BurnedMarshal**
Created a new swagger file for authentication routes.
Was added missing `/logout` specification
Was fixed the specification of `/user-identity` API because the base path was wrong compared with the real implementation.

### CODE REVIEW

File: api_auth.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Authentication API
+  description: Collection of enpoints to interact with user auth session.
```
Comment (gunzip) [inline api_auth.yaml:5]:
```suggestion
  description: Collection of endpoints to interact with user's auth session.
```

**Review by gunzip — CHANGES_REQUESTED** [review]

File: api_auth.yaml (hunk 1 above)
Comment (BurnedMarshal) [reply api_auth.yaml]: Fixed in c2e442d

File: api_auth.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  - https
+security:
+  - Bearer: []
+paths:
+  "/logout":
+    x-swagger-router-controller: AuthenticationController
+    post:
+      operationId: logout
+      summary: Execute the logout
+      description: Delete the user active session and the token is invalidated.
```
Comment (gunzip) [inline api_auth.yaml:18]:
```suggestion
      description: Delete user's active session and tokens.
```

File: api_auth.yaml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      responses:
+        "200":
+          description: Logout succeded
```
Comment (gunzip) [inline api_auth.yaml:21]:
```suggestion
          description: Logout succeeded
```

File: api_auth.yaml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+            application/json:
+              message: "ok"
+        "400":
+          description: Bad request
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Bearer token null or expired.
+        "500":
+          description: There was an error deleting the user active session.
```
Comment (gunzip) [inline api_auth.yaml:34]:
```suggestion
          description: There was an error deleting user's session.
```

File: api_auth.yaml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+          schema:
+            $ref: "#/definitions/ProblemJson"
+  "/user-identity":
+    x-swagger-router-controller: AuthenticationController
+    get:
+      operationId: getUserIdentity
+      summary: Get stored user identity
```
Comment (gunzip) [inline api_auth.yaml:41]:
```suggestion
      summary: Get stored user's identity
```

File: api_auth.yaml [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+      description: Returns the user identity stored during the login phase
```
Comment (gunzip) [inline api_auth.yaml:42]:
```suggestion
      description: Returns the user's identity stored during the login phase
```

File: api_auth.yaml [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+              spid_email: "preferred@example.com"
+              spid_mobile_phone: "1234567890"
+        "400":
+          description: Bad request
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Bearer token null or expired.
+        "500":
+          description: There was an error in retrieving the user profile.
```
Comment (gunzip) [inline api_auth.yaml:62]:
```suggestion
          description: There was an error retrieving user's profile.
```

**Pull request merged**